lib/assets/javascripts/turbograft/turbohead.coffee

```diff
@@ -14,6 +22,10 @@ class window.TurboHead
       .filter(attributeMatches('nodeName', 'LINK'))
       .filter(noAttributeMatchesIn('href', @activeAssets))
 
+  @reset: ->
```

Comment:
Yes, agree that the domain model needs to be adjusted to the new requirements.  As we discussed in Slack, this isn't a refactoring PR.